travis.yml: error setting $VERSION

Introduced in #76.

The build job is adding all the maven "Download..." output to the value of `VERSION`. That is likely why this build step later fails with `[ERROR] No plugin found for prefix 'https'`.

--- a/.travis.yml
+++ b/.travis.yml
@@ -35,7 +35,7 @@ jobs:
     - openssl aes-256-cbc -K $encrypted_af28e35ab9a0_key -iv $encrypted_af28e35ab9a0_iv
       -in release/codesigning.asc.enc -out release/codesigning.asc -d
     - gpg --fast-import release/codesigning.asc
-    - export VERSION=$(mvn help:evaluate -Dexpression=project.version | grep -v '^\[' | sed "s/SNAPSHOT/rc$TRAVIS_BUILD_ID/")
+    - export VERSION=$(mvn help:evaluate -Dexpression=project.version | grep -v '^\[' | grep -v '^Download' | sed "s/SNAPSHOT/rc$TRAVIS_BUILD_ID/")
     - echo "Candidate Version is $VERSION"
     - mvn versions:set -DnewVersion=$VERSION
     - mvn --settings release/mvnsettings.xml -P deploy -DskipTests=true clean deploy

